common/customizable_localization/00_divinity_custom_loc.txt

```diff
@@ -567,56 +567,495 @@ GetActualBishopTitle = {
+		localization_key = councillor_court_chaplain_nerubian_religion_duchy_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:nerub_group
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_nerubian_religion_county
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:nerub_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_nerubian_religion_county_female
+	}
+	# Pandaren
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			religion = religion:pandaren_religion_group
+		}
+		localization_key = councillor_court_chaplain_pandaren_religion_kingdom
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			religion = religion:pandaren_religion_group
+		}
+		localization_key = councillor_court_chaplain_pandaren_religion_duchy
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:pandaren_religion_group
+		}
+		localization_key = councillor_court_chaplain_pandaren_religion_county
+	}
+	# Secular
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			religion = religion:secular_group
+		}
+		localization_key = councillor_court_chaplain_secular_religion_kingdom
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			religion = religion:secular_group
+		}
+		localization_key = councillor_court_chaplain_secular_religion_duchy
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:secular_group
+		}
+		localization_key = councillor_court_chaplain_secular_religion_county
+	}
+	# Void
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			religion = religion:void_group
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_void_religion_kingdom
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_kingdom
+			religion = religion:void_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_void_religion_kingdom_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			religion = religion:void_group
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_void_religion_duchy
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier >= tier_duchy
+			religion = religion:void_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_void_religion_duchy_female
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:void_group
+			is_female = no
+		}
+		localization_key = councillor_court_chaplain_void_religion_county
+	}
+	text = {
+		trigger = {
+			liege.highest_held_title_tier < tier_duchy
+			religion = religion:void_group
+			is_female = yes
+		}
+		localization_key = councillor_court_chaplain_void_religion_county_female
 	}
```

Comment:
I would make it family's default for future use.

Reply:
done